server/controllers/articleController.js

```diff
@@ -0,0 +1,40 @@
+import uniqid from 'uniqid';
+import articles from '../models/articles';
+
+
+/**
+ * @class ArticleController
+ * @exports ArticleController
```

Comment:
Hello Soji, would you mind renaming this to just `article.js`, as it is already in the controller folder?